src/onboarding/onboarding-config.ts

```diff
@@ -0,0 +1,57 @@
+import {StaticOnboardingSection} from '@atb/onboarding';
+
+export const staticOnboardingSectionsInPrioritizedOrder: StaticOnboardingSection[] =
```

Comment:
I like the idea of define the screens to show in the onboard, need to have a more deep look and see how it works, It's nice to see it is more dynamic and is a good approach, I will suggest in the future add more attributes like next Screen, order and parameters to be expected to receive. and the possibility to configure it from the remote Config, but of course might not be part of this PR.

Reply:
The config format can be adapted to new feature requests for sure :+1: